tower_cli/models/base.py

```diff
@@ -733,7 +734,7 @@ def list(self, all_pages=False, **kwargs):
         # Alter the "next" and "previous" to reflect simple integers,
         # rather than URLs, since this endpoint just takes integers.
         for key in ('next', 'previous'):
-            if not response[key]:
+            if not response.get(key):
```

Comment:
`response` is a dict, so it would be a `KeyError`.  `dict.get` falls back to `None`, so I think this should work: https://docs.python.org/2/library/stdtypes.html#dict.get

`python -c "print {}.get('missing') is None"`